node-deb

```diff
@@ -418,9 +418,9 @@ log_debug "The package maintainer has been set to: $package_maintainer"
 if [ -z "$package_dependencies" ]; then
   package_dependencies=$(jq -r '.node_deb.dependencies' package.json)
   if [[ "$package_dependencies" == 'null' ]]; then
-    package_dependencies="nodejs"
+    package_dependencies="nodejs, sudo"
```

Comment:
Currently this breaks Travis because there's no sudo in some of the containers. I just pushed a change to the test script so that you can modify the docker images and then Travis will use them on the PR.